symengine/__init__.py

```diff
@@ -9,7 +9,7 @@
         UndefFunction, Function, FunctionSymbol as AppliedUndef,
         have_numpy, true, false, Equality, Unequality, GreaterThan,
         LessThan, StrictGreaterThan, StrictLessThan, Eq, Ne, Ge, Le,
-        Gt, Lt, GoldenRatio, Catalan, EulerGamma, Dummy, perfect_power,
+        Gt, Lt, golden_ratio, catalan, eulergamma, Dummy, perfect_power,
```

Comment:
Should we just include the classes here? Instances would anyways be available through the Singleton class.